### ISSUE TITLE
Reproduction of the results of tiered-imagenet

### ISSUE THREAD

**Issue by RongKaiWeskerMA** [icoz69/DeepEMD#37]
Hi, thanks for the contribution to few-shot learning community. It is indeed interesting work!

I am just wondering how to get the exact results you report in the paper for tiered-imagenet. Maybe not the exact results but close results. I can get 73% accuracy on 1 shot using the hyperparameters given in the readme file with 600 randomly sampled episodes, however, only 70% with 5,000 episodes.  Thanks!

**Comment by icoz69** [comment]
hi generally the parameters to adjust are lr scheduler, such as decay step and decay strength. I have recently found that using cosine lr scheduler without annealing is good as no hyper-parameters about lr scheduler is needed any more, and you only need to find a good start lr.
